components/HistoryTable.qml

```diff
@@ -266,6 +266,45 @@ ListView {
 
                 }
             }
+            Item { //separator
+                width: 100
+                height: 14
+            }
+            // -- "Received by" title
+            Text {
+                anchors.bottom: parent.bottom
+                font.family: "Arial"
+                font.pixelSize: 12
+                color: "#535353"
+                text: (isOut ? qsTr("Spent from:") : qsTr("Received by:")) + translationManager.emptyString
+            }
+            Item { //separator
+                width: 5
+                height: 14
+            }
+            // -- "Index" value
+            Text {
+                anchors.bottom: parent.bottom
+                font.family: "Arial"
+                font.pixelSize: 13
```

Comment:
Perhaps make all, or almost all constant integer values end with `* scaleRatio` ? I might be reviewing my own code here, in which case I have forgotten it :)

Reply:
Does this apply to every single place where constant integers are used? There are quite many, like `rowSpacing, width, height, rightMargin, fontSize, ...` in addition to `font.pixelSize`.

Reply:
I remember @Jaqueeee told me to do that for all constant integers IIRC

Reply:
> we can always change it later when I focus on the mobile part (soon).

I bet that's better since doing the mobile layout right is a big deal on its own.